hana/lib/cql-functions.js

```diff
@@ -0,0 +1,29 @@
+const StandardFunctions = {
```

Comment:
I think we need to move the mapping for the [`HanaFunctions` ](https://github.com/cap-js/cds-dbs/blob/afda1703c370d7aded6355856f4e906a4562b040/db-service/lib/cql-functions.js#L89) in `db-service/lib/cql-functions.js` to the sqlite service so that the native functions will actually work on HANA